web/src/user_profile.js

```diff
@@ -343,6 +345,17 @@ function initialize_user_type_fields(user) {
     }
 }
 
+export function show_user_profile_access_error_modal() {
+    $("body").append(render_profile_access_error_model());
```

Comment:
WIll this leak copies of this modal if you access several invalid user IDs in sequence? Or does `autoremove` handle that properly?

(It would be nice if the link between the element created here and the one accessed in `modals.open` were more obvious if so; it looks to me like it's entirely through the ID in the template being the same?)

A comment here would be helpful.

Reply:
I have verified that the `auto-remove` ensures the removal of the modal from the DOM even when accessing several invalid user IDs in sequence.

About the connection between the element and the one being passed in the `models.open`, both are linked through the modal ID. To make it look obvious, I will add the comment "This opens the model, referencing it by it's ID ('profile_access_error_model)".